src/vorta/store/connection.py

```diff
@@ -62,12 +63,12 @@ def init_db(con=None):
     # Delete old log entries after 6 months.
     # The last `create` command of each profile must not be deleted
     # since the scheduler uses it to determine the last backup time.
-    last_backups_per_profile = (
+    last_backups_per_profile = Tuple(
```

Comment:
I don't think this is proper use of type hints. Shouldn't it be the following instead?
```suggestion
    last_backups_per_profile : Tuple = 
```